constance/backends/database.py

```diff
@@ -24,24 +25,25 @@ def __init__(self):
         if self._model._meta.app_config is None:
             raise ImproperlyConfigured(
                 "The constance.backends.database app isn't installed "
-                "correctly. Make sure it's in your INSTALLED_APPS setting.")
+                "correctly. Make sure it's in your INSTALLED_APPS setting."
+            )
 
         if settings.DATABASE_CACHE_BACKEND:
             self._cache = caches[settings.DATABASE_CACHE_BACKEND]
             if isinstance(self._cache, LocMemCache):
                 raise ImproperlyConfigured(
-                    "The CONSTANCE_DATABASE_CACHE_BACKEND setting refers to a "
+                    'The CONSTANCE_DATABASE_CACHE_BACKEND setting refers to a '
                     "subclass of Django's local-memory backend (%r). Please "
-                    "set it to a backend that supports cross-process caching."
-                    % settings.DATABASE_CACHE_BACKEND)
+                    'set it to a backend that supports cross-process caching.' % settings.DATABASE_CACHE_BACKEND
+                )
```

Comment:
Should we take the opportunity to migrate this to a f-string?

Reply:
We should. There are linters for that in ruff. But to reduce complexity of reviewing process I've moved this out of scope of this PR.

Reply:
Looks like the start of a great refresh :pray:

Reply:
OK. than go next 😄